Makefile

```diff
@@ -285,7 +285,9 @@ endif
 gardener-up gardener-down gardenlet-kind2-up gardenlet-kind2-down: export SKAFFOLD_LABEL = skaffold.dev/run-id=gardener-local
 
 gardener-up: $(SKAFFOLD) $(HELM)
-	$(SKAFFOLD) run
+	kubectl apply -k $(REPO_ROOT)/example/gardener-local/registry --server-side
+	kubectl wait --for=condition=available deployment -l app=registry -n registry --timeout=2m
+	SKAFFOLD_DEFAULT_REPO=localhost:5001 $(SKAFFOLD) run
```

Comment:
Instead of setting `push=false` in `skaffold.yaml`, let's rather set `SKAFFOLD_PUSH=false` here, as it somehow belongs together with `SKAFFOLD_DEFAULT_REPO`.

Reply:
Now that we will reuse the registry also for the kind2 cluster, I think it's actually wrong to set `push=false` in the `kind2` profile. We should push in both cases to make sure that new images are available from the registry also when `make gardenlet-kind2-up` is called and local changes were made. Will change it accordingly.